website/_includes/release-notes-feed.html

```diff
@@ -0,0 +1,77 @@
+{% set is_beta = channel_name == "Beta" %}
+<p>{{ _('Check out the notes below for this version of Thunderbird. As always, you’re encouraged to <a href="%(feedback)s">tell us what you think</a>, or <a href="%(bugzilla)s">file a bug in Bugzilla</a>')|format(feedback='https://support.mozilla.org/questions/new/thunderbird', bugzilla='https://bugzilla.mozilla.org/') }}</p>
```

Comment:
For the feed, lets remove this line. At the bottom after the notes, put the "As always... Bugzilla" links.